beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,491 @@ title: Quick Start
+```js {5}
+function MyButton() {
+  const [count, setCount] = useState(0);
 
-</LearnMore>
+  function handleClick() {
+    setCount(count + 1);
+  }
 
-### Chapter 3 overview: Managing state {/*chapter-3-overview-managing-state*/}
+  return (
+    <button onClick={handleClick}>
+      Clicked {count} times
+    </button>
+  );
+}
+```
 
-You'll often face a choice of _what exactly_ to put into state. Should you use one state variable or many? An object or an array? How should you [structure your state](/learn/choosing-the-state-structure)? The most important principle is to **avoid redundant state**. If some information never changes, it shouldn't be in state. If some information is received from parent by props, it shouldn't be in state. And if you can compute something from other props or state, it shouldn't be in state either!
+React will call your component function again. This time, `count` will be `1`. Then it will be `2`. And so on.
 
-For example, this form has a **redundant** `fullName` state variable:
+If you render the same component multiple times, each will get its own state. Try clicking each button separately:
 
 <Sandpack>
 
 ```js
 import { useState } from 'react';
 
-export default function Form() {
-  const [firstName, setFirstName] = useState('');
-  const [lastName, setLastName] = useState('');
-  const [fullName, setFullName] = useState('');
+function MyButton() {
+  const [count, setCount] = useState(0);
 
-  function handleFirstNameChange(e) {
-    setFirstName(e.target.value);
-    setFullName(e.target.value + ' ' + lastName);
+  function handleClick() {
+    setCount(count + 1);
   }
 
-  function handleLastNameChange(e) {
-    setLastName(e.target.value);
-    setFullName(firstName + ' ' + e.target.value);
-  }
+  return (
+    <button onClick={handleClick}>
+      Clicked {count} times
+    </button>
+  );
+}
 
+export default function MyApp() {
   return (
-    <>
-      <h2>Let’s check you in</h2>
-      <label>
-        First name:{' '}
-        <input
-          value={firstName}
-          onChange={handleFirstNameChange}
-        />
-      </label>
-      <label>
-        Last name:{' '}
-        <input
-          value={lastName}
-          onChange={handleLastNameChange}
-        />
-      </label>
-      <p>
-        Your ticket will be issued to: <b>{fullName}</b>
-      </p>
-    </>
+    <div>
+      <h1>Counters that update separately</h1>
+      <MyButton />
+      <MyButton />
+      <MyButton />
+    </div>
   );
 }
 ```
 
 ```css
-label { display: block; margin-bottom: 5px; }
+button {
+  display: block;
+  margin-bottom: 5px;
+}
 ```
 
 </Sandpack>
 
-You can remove it and simplify the code by calculating `fullName` while the component is rendering:
+Functions starting with `use` are called Hooks. `useState` is a built-in Hook provided by React. You can find other built-in Hooks in the [React API reference](/reference). You can also write your own Hooks by combining the existing ones.
```

Comment:
**problem**: It feels like this paragraph lacks context. 

**possible solution**: add a header ("About hooks") or place this paragraph a visually-different section (for example some "NOTE" section)